src/modules/bulletproofs/bulletproofs_util.h

```diff
@@ -20,6 +20,22 @@ static void secp256k1_bulletproofs_serialize_points(unsigned char *output, const
     secp256k1_fe_get_b32(&output[33], &rpt->x);
 }
 
+/* Initializes SHA256 with fixed midstate. This midstate was computed by applying
+ * SHA256 to SHA256("Bulletproofs/commitment")||SHA256("Bulletproofs/commitment"). */
```

Comment:
Lol, I can't reproduce this :P . See what I did to reproduce this in this commit. https://github.com/sanket1729/rust-bitcoin/commit/5b289a900f5263f5f185cf1afb1a638c34098849 . Even if this is displayed reverse this is not matching 

The tagged hash code should 100% be correct because it computes the values for taproot hashes that we have tested on bitcoin network.

Reply:
🤦 Sorry I missed this during review

Reply:
I get the same. "1c1a5ea2a673f92997c73f92088493dec79ad232540e40b470749a0d79a8b650" which is just displayed reverse

Reply:
I agree. thanks for catching this. I'm not sure where my original values came from.